### ISSUE TITLE
Getting an error starting pytyle1x

### ISSUE THREAD

**Issue by jamiefraser77** [programical/pytyle1x#10]
I'm getting an error message after installing with pip3 and copying the example `pytylerc` into `.config`:

```
jam@jam-ncase:~$ pytyle1x
4/6/2021 at 12:46:35:    PyTyle started.
4/6/2021 at 12:46:35:    Fatal error!
4/6/2021 at 12:46:35:    Traceback (most recent call last):
  File "/home/jam/.local/bin/pytyle1x", line 41, in <module>
    Window.load_new_windows()
  File "/home/jam/.local/lib/python3.8/site-packages/PyTyle/Window.py", line 36, in load_new_windows
    Window.load_window(win)
  File "/home/jam/.local/lib/python3.8/site-packages/PyTyle/Window.py", line 50, in load_window
    attrs = PROBE.get_window_by_id(window_id)
  File "/home/jam/.local/lib/python3.8/site-packages/PyTyle/Probe.py", line 376, in get_window_by_id
    return self.get_window(win)
  File "/home/jam/.local/lib/python3.8/site-packages/PyTyle/Probe.py", line 283, in get_window
    windesk = win.get_full_property(
AttributeError: 'NoneType' object has no attribute 'value'

```

**Comment by programical** [comment]
What window manager or desktop environment are you using PyTyle with?

**Comment by jamiefraser77** [comment]
Using Ubuntu Budgie 20.04, the window manager is mutter.

**Comment by programical** [comment]
Mutter should be ok, since it apparently supports EWMH. Please post the output of `echo $XDG_SESSION_TYPE`.

**Comment by jamiefraser77** [comment]
`X11`

**Comment by programical** [comment]
Confirmed. The window manager must be the problem, because when I replaced it with xfwm4, PyTyle started working.

**Comment by programical** [comment]
Fixed in [v1.0.1](https://github.com/programical/pytyle1x/releases/tag/v1.0.1)
